gulpfile.js

```diff
@@ -58,7 +60,9 @@ const LEGIBLE_NAMES = [
   { identifier: 'None', legibleName: 'none' },
 ];
 
-const format = s => s[0].toUpperCase() + _.camelCase(s.substring(1));
+function capitalizeFirstLetter(string) {
```

Comment:
This isn't needed because `capitalize` from `lodash` can be used

Reply:
Further why are we changing this at all? It doesn't seem like it is needed

Reply:
I used it in a helper method (where I transform the icon names into constants), and the same logic was there in two different methods, so I just used `capitalizeFirstLetter` for both

Reply:
capitalizeFirstLetter seemed to be a better suited name for the use case, but I rename it back if you want to